pkgs/development/python-modules/datrie/default.nix

```diff
@@ -17,17 +19,24 @@ buildPythonPackage rec {
     hash = "sha256-UlsI9jjVz2EV32zNgY5aASmM0jCy2skcj/LmSZ0Ydl0=";
   };
 
+  patches = [
```

Comment:
The only drawback of integrating this non-merged patch is that it could make the maintenance of this package a bit harder if there is - for instance - a diff conflict in the future.

Did you consider adding yourself in the datrie maintainer list?

Reply:
Thanks for reviewing, guys! I'll change this patch to a substituteInPlace or sed.

I do tend to submit even these minor changes upstream and then pull them in as patches as a form of documentation. If a conflict occurs, I figure it's a good thing since it calls to attention that something changed. I understand opinions can differ here :)

Reply:
I updated this PR to use `postPatch` instead of a patch. Kept the wheel change in for now.

Reply:
> tjni is doing work on the python infrastructure itself and fixing packages 

Ok, i didn't know and i then understand @tjni  won't maintain all impacted packages ;)